src/index.js

```diff
@@ -244,12 +245,6 @@ function addToProto (ctx, obj, proto, disablePromises) {
   }
 }
 
-const cozy = new Cozy()
-
-export default cozy
-export { Cozy, LocalStorage, MemoryStorage }
-
-if ((typeof window) !== 'undefined') {
-  window.cozy = cozy
-  window.Cozy = Cozy
-}
+module.exports = new Client()
+module.exports.LocalStorage = LocalStorage
+module.exports.MemoryStorage = MemoryStorage
```

Comment:
We can't, as webpack expect `module.exports` (so, a commonjs format) to build proper UMD. Else, it won't output a `root['cozy']['client'] = Client` but `root['cozy']['client'] = {default: Client}` which isn't what we expect :wink: